building/tooling/best-practices.md

```diff
@@ -53,7 +53,7 @@ Tooling runs as one-off, short-lived Docker container:
 3. The Docker container is destroyed
```

Comment:
```suggestion
3. The Docker container is destroyed.
```

Reply:
Hmm, why is a period needed here?

Reply:
Ah, mistake on my part on mobile.

Reply:
I can't add a comment to it, but L49 should read "Tooling runs as **a** one-off, short-lived Docker container:"

Reply:
Good catch. Fixed.

Reply:
L140 should read 'Distributions that **use** the `apk` package manager (such as Alpine) should use the `--no-cache` flag when using `apk add` to install packages:'

Reply:
L217 should read 'Its [Dockerfile](https://github.com/exercism/ruby-test-runner/blob/e57ed45b553d6c6411faeea55efa3a4754d1cdbf/Dockerfile) starts with a stage (given the name `build`) that **installs** those packages (via `apk add`) and then installs the **libraries** (via `bundle install`):'